### PR TITLE
BitConnect added

### PR DESCRIPTION
**Pull request by VladimirLevv**

### CODE REVIEW

File: projects/bitconnect.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+### BitConnect
+![BitConnect logo](https://bitcointalk.org/useravatars/avatar_919758.png)
+ 1. Website: [BitConnect (BCC)](https://bitconnectcoin.co/)
+ 2. Founders: Group of friends from Toronto
```
Comment (akhavr) [inline projects/bitconnect.md]: Who are the founders?  Any links?

Comment (VladimirLevv) [inline projects/bitconnect.md]: ok, searching

Comment (VladimirLevv) [inline projects/bitconnect.md]: searching for the info

**Comment by akhavr** [comment]
додай Ken Fitzsimmons на People

**Pull request merged**